src/FormControls/RadioButton.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default class RadioButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getInputDOMNode() {
+    return React.findDOMNode(this.refs.input);
+  }
+
+  getValue() {
+    return this.props.value;
+  }
+
+  getChecked() {
+    return this.getInputDOMNode().checked;
+  }
+
+  renderLabel(input) {
+    if (!this.props.label) {
+      return input;
+    }
+
+    const {id, label} = this.props;
+
+    const classes = {
+      'radio': !this.props.inline,
+      'radio-inline': this.props.inline
+    };
+
+    return (
+      <label className={classNames(classes, this.props.labelClassName)} htmlFor={id}>
```

Comment:
In the past I have had trouble adding the class `radio` directly to the label element. tbs docs add it to a surrounding div when used in a `FormGroup` in a horizontal form, otherwise something is off about the margins. We should confirm that that isn't a problem here. According to our docs we follow the same behavior for checkbox